readme.md

```diff
@@ -40,4 +42,4 @@ Project has a module maven structure and consists of 3 modules:
 
 **Examples**
 
-To have a sidechain node without any custom business logic run [Simple App](examples/simpleapp/readme.md) example.
+To have a sidechain node without any custom business logic run [Simple App](examples/simpleapp/mc_sc_workflow_example.md) example.
```

Comment:
Why not to refer to `examples/simpleapp/readme.md` where the first Simple-app description is present?

Reply:
changed to have both references